python/docs/examples.rst

```diff
@@ -0,0 +1,203 @@
+A quick guide
+=============
+
+This is a quick guide to get you started with dlisio. Note that all classes,
+functions are more thoroughly documented under :ref:`The Library`. Please refer
+there for more information about them.
+
+The same documentation is also available directly in your favorite python
+interpreter and in the unix console, just type :code:`help(function)` or  :code:`pydoc
+function`, respectably. In the interpreter, help can be used directly on
+class instances. E.g: :code:`help(frame)` or :code:`help(frame.curves)`
+
+Opening files
+-------------
+
+Load all :ref:`Logical files`:
+
+.. code-block:: python
+    
+    >>> with dlisio.load(filename) as files:
+    ...     for f in files:
+    ...         pass
+
+The returned :code:`files` can be iterated over and operations can be applied
+to each logical file.
+
+If you only want to work with one logical file at the time,
+:py:func:`dlisio.load()` support automatic unpacking of logical file. The
+following syntax unpacks the first logical file into :code:`f` and stores the
+rest (0-n) logical files into :code:`*tail`.
+
+
+.. code-block:: python
+
+    >>> with dlisio.load(filename) as (f, *tail):
+    ...     pass
+
+Or, if the number of logical files is known:
+
+.. code-block:: python
+
+    >>> with dlisio.load(filename) as (f1, f2, f3):
+    ...     pass
+
+When a file is loaded, you can output some basic information about the physical
+file:
+
+.. code-block:: python
+
+    >>> import dlisio
+    >>> with dlisio.load(filename) as files:
+    ...     files.describe()
+    ----------------------
+    Batch of Logical Files
+    ----------------------
+    Number of Logical Files : 3
+
+    Description : dlis(DDBC1)
+    Frames      : 0
+    Channels    : 0
+    
+    Description : dlis(DDBC2)
+    Frames      : 2
+    Channels    : 22
+    
+    Description : dlis(DDBC3)
+    Frames      : 2
+    Channels    : 160
+
+Or about a logical file:
+
+.. code-block:: python
+
+    >>> import dlisio
+    >>> with dlisio.load(filename) as (f, *tail):
+    ...     f.describe()
+    ------------
+    Logical File
+    ------------
+    Description  : dlis(MSCT_200LTP)
+    Frames       : 2
+    Channels     : 104
+    Object count : 570
+
+    Known objects
+    --
+    FILE-HEADER             : 1
+    ORIGIN                  : 1
+    CALIBRATION-COEFFICIENT : 24
+    CHANNEL                 : 104
+    FRAME                   : 2
+
+    Unknown objects
+    --
+    440-CHANNEL        : 95
+    440-OP-CORE_TABLES : 250
+    440-OP-CHANNEL     : 93
+
+Accessing objects
+-----------------
+
+Think of :ref:`Logical files` as pools of objects with different types.  All
+objects of a type can be reached by name:
```

Comment:
Wasn't clear to me what you meant, so maybe
All objects of a type can be reached by name, for example channels or coefficients: